src/PlusDataCollection/vtkPlusGenericSerialDevice.cxx

```diff
@@ -56,6 +56,8 @@ vtkPlusGenericSerialDevice::vtkPlusGenericSerialDevice()
   : Serial(new SerialLine())
   , SerialPort(1)
   , BaudRate(9600)
+  , DTR(false)

```

Comment:
If this is not descriptive enough, I guess CurrentDTR is the best name for it. But that would screw up `vtkGetMacro(RTS, bool);`. Do you still want me to change it?